readme.md

```diff
@@ -272,6 +275,39 @@ Yields:
 <h1 id="xxx"><a aria-hidden="true" tabindex="-1" href="#xxx"><span class="visually-hidden">Read the “xxx” section</span><span class="icon icon-link" aria-hidden="true"></span></a>xxx</h1>
 ```
 
+### Example: content from `fromHtml` builder
+
+The following example passes `content` as a function, to generate an accessible
+description of each link.
+
+```js
+import {rehype} from 'rehype'
+import rehypeAutolinkHeadings from 'rehype-autolink-headings'
+import {fromHtml} from "hast-util-from-html"
+
+main()
+
+async function main() {
+  const file = await rehype()
+    .data('settings', {fragment: true})
+    .use(rehypeAutolinkHeadings, {
+      content: fromHtml(
+        `<svg height="10" width="10"><circle cx="5" cy="5" r="5" fill="black" /></svg>`,
+        {space: "svg", fragment: true}
+      ),
```

Comment:
```suggestion
      content: fromHtmlIsomorphic(
        '<svg height="10" width="10"><circle cx="5" cy="5" r="5" fill="black" /></svg>',
        {fragment: true}
      ).children
```

1. If you are going to include an HTML parser, `fromHtmlIsomorphic` is a better idea
2. Match code style
3. Use `children` from the root, so as not to include a `root`, which shouldn’t be done, this still allows for the theoretical case where a doctype is parsed by `fromHtml`, so the types don’t match, but that practically does not occur